content/blobstore-tls.md

```diff
@@ -0,0 +1,83 @@
+!!! note
+    Applicable for director version 266.2.0+
+
+    Linux stemcells v3586.5+ (agent 2.83.0)
+
+    Windows 2016 stemcell v1709.9+, Windows 2012R2 v1200.20+ (agent 2.110.0)
```

Comment:
also for windows, wondering what we relied on to specify this stemcell version ?

Reply:
https://github.com/cloudfoundry/bosh-agent/commit/55990481ff13b5991bcec77f6e707809b2aa14f9 this commit & consulting with the windows bosh team.